userenvs/stream8-flexran.json

```diff
@@ -0,0 +1,94 @@
+{
+  "workshop": {
+    "schema": {
+      "version": "2022.07.25"
+    }
+  },
+  "userenv": {
+    "name": "stream8-flexran",
+    "label": "flexRAN Stream 8",
+    "origin": {
+      "image": "quay.io/xxx/yyy",
+      "tag": "zzz"
```

Comment:
Is the implication here that the user is supposed to update these fields prior to using this?

Reply:
Yes.  Intel seems to crank out 2,3 FlexRAN releases a year.  We build images from these releases as needed. So we are on a moving train.

Reply:
My concern, if you want to call it that, is not so much that the "train" is moving (you could say that about all of the userenvs really) but that it needs to be well documented somewhere/somehow that there is this expectation for this one userenv.  Maybe add a `README.md` file in the `stream8-flexran` directory that documents how this userenv is expected to be used?